bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/handler/InverterHandler.java

```diff
@@ -109,6 +108,45 @@ protected void handlePolledData(ModbusRegisterArray registers) {
         updateState(channelUID(GROUP_AC_GENERAL, CHANNEL_AC_LIFETIME_ENERGY),
                 getScaled(block.acEnergyLifetime, block.acEnergyLifetimeSF, WATT_HOUR));
 
+        // DC General group
+        updateState(channelUID(GROUP_DC_GENERAL, CHANNEL_DC_CURRENT),
+                getScaled(block.dcCurrent, block.dcCurrentSF, AMPERE));
+        updateState(channelUID(GROUP_DC_GENERAL, CHANNEL_DC_VOLTAGE),
+                getScaled(block.dcVoltage, block.dcVoltageSF, VOLT));
+        updateState(channelUID(GROUP_DC_GENERAL, CHANNEL_DC_POWER), getScaled(block.dcPower, block.dcPowerSF, VOLT));
```

Comment:
you are using unit volts for power?

Reply:
bummer. Of course not. Fixed this, thanks for pointing it out.